ni_measurementlink_service/session_management.py

```diff
@@ -305,40 +316,44 @@ def _get_stub(self) -> session_management_service_pb2_grpc.SessionManagementServ
     def reserve_session(
         self,
         context: PinMapContext,
-        pin_or_relay_names: Optional[Iterable[str]] = None,
+        pin_or_relay_names: Union[str, Iterable[str], None] = None,
         instrument_type_id: Optional[str] = None,
-        timeout: Optional[float] = None,
+        timeout: Optional[float] = 0.0,
     ) -> SingleSessionReservation:
         """Reserve a single session.
 
         Reserve the session matching the given pins, sites, and instrument type ID and return the
         information needed to create or access the session.
 
         Args:
-            context (PinMapContext): Includes the pin map ID for the pin map in the Pin Map Service,
+            context: Includes the pin map ID for the pin map in the Pin Map Service,
                 as well as the list of sites for the measurement.
 
-            pin_or_relay_names (Iterable[str]): List of pins, pin groups, relays, or relay groups to
-                use for the measurement. If unspecified, reserve sessions for all pins and relays in
-                the registered pin map resource.
+            pin_or_relay_names: One or multiple pins, pin groups, relays, or relay groups to use
+                for the measurement.
 
-            instrument_type_id (str): Instrument type ID for the measurement. If unspecified,
-                reserve sessions for all instrument types connected in the registered pin map
+                If unspecified, reserve sessions for all pins and relays in the registered pin map
                 resource.
 
+            instrument_type_id: Instrument type ID for the measurement.
+
+                If unspecified, this method reserve sessions for all instrument types connected
+                in the registered pin map resource.
+
                 For NI instruments, use instrument type id constants, such as
                 :py:const:`INSTRUMENT_TYPE_NI_DCPOWER` or :py:const:`INSTRUMENT_TYPE_NI_DMM`.
 
                 For custom instruments, use the instrument type id defined in the pin map file.
 
-            timeout (float): Timeout in seconds. Allowed values,
-                0 (non-blocking, fails immediately if resources cannot be reserved),
-                -1 or negative (infinite timeout), or
-                any positive numeric value (wait for that number of second).
+            timeout: Timeout in seconds.
```

Comment:
Milliseconds

Reply:
Correction: Change 'milliseconds' below to seconds.

Reply:
Oops, I updated this from the .proto file and didn't notice that. Thanks.